src/EditorFeatures/Core/Options/CompletionViewOptions.cs

```diff
@@ -20,5 +20,9 @@ internal sealed class CompletionViewOptions
 
         public static readonly PerLanguageOption2<bool> BlockForCompletionItems =
             new("CompletionOptions_BlockForCompletionItems", defaultValue: true);
+
+        public static readonly Option2<bool> HideExpandedItemsForShortFilterTextFeatureFlag =
```

Comment:
@tmat There has been changes around featrueflag storage, etc. just wanted to make sure I'm doing this correctly.